com.atlauncher.ATLauncher.yml

```diff
@@ -0,0 +1,55 @@
+# Credits to github/nickavem for the script, Doomsdayrs for edits
+
+app-id: com.atlauncher.ATLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk8
+command: atlauncher.sh
+build-options:
+    env:
+        PATH: /app/jre/bin:/usr/bin
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --socket=x11
+    - --share=ipc
+    - --device=all
+    - --socket=pulseaudio
+    - --share=network
+modules:
```

Comment:
As mentioned elsewhere, these modules can be really merged into one.

Reply:
Unsure why this has been marked as resolved.

Reply:
There does not seem to be any reason aside from style preference.